pyproject.toml

```diff
@@ -59,6 +59,7 @@ wheel.install-dir = "cmake/data"
 
 [tool.setuptools_scm]
 write_to = "src/cmake/_version.py"
+write_to_template = "version = '{version}'"
```

Comment:
Are these fixes specific to python 3.5 and 2.7 ?

Reply:
yes. I don't know if it's required for python 3.6 or not.

Reply:
```suggestion
# Setuptools-scm includes type annotations in the default template
write_to_template = "version = '{version}'"
```